types/src/message.rs

```diff
@@ -67,6 +67,79 @@ impl<
     }
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+#[serde(bound(deserialize = ""))]
+/// Votes sent by consensus messages.
+pub enum Vote<TYPES: NodeType, LEAF: LeafType<NodeType = TYPES>> {
```

Comment:
👍  We should have an issue noting that this is a stopgap until the traits refactoring is complete, and link it in a comment here.

Reply:
Oh this is also something added by the voting interface PR that's already merged, so not part of this PR actually. I just created an issue: https://github.com/EspressoSystems/HotShot/issues/856 for this. I could link it in the next PR for fixing the next leader logic (https://github.com/EspressoSystems/HotShot/issues/854), so Brendon could merge this PR sooner.